src/TraitsV2-Tests/T2AbstractTest.class.st

```diff
@@ -62,8 +62,8 @@ T2AbstractTest >> setUp [
 
 { #category : #running }
 T2AbstractTest >> tearDown [
-	createdClasses reverse do:#removeFromSystem.
+	createdClasses reverse do: #removeFromSystem.
```

Comment:
`#reverseDo:` is normally better from a performance point of view since it does not create intermediate collection.

```suggestion
	createdClasses reverseDo: #removeFromSystem.
```